src/ymfm.h

```diff
@@ -295,23 +295,23 @@ struct ymfm_output
 	// clear all outputs to 0
 	ymfm_output &clear()
 	{
-		for (int index = 0; index < NumOutputs; index++)
+		for (size_t index = 0; index < NumOutputs; index++)
 			data[index] = 0;
```

Comment:
Would prefer in general to use uint32_t instead of size_t here  since size_t varies by platform.

Reply:
(And in general throughout this PR please use uint32_t instead of size_t unless a specific API is returning a size_t.)